legal_db/templates/legal_db/base.html

```diff
@@ -7,6 +7,8 @@
     <meta charset="utf-8"/>
     <meta content="width=device-width, initial-scale=1" name="viewport"/>
     <meta name="description" content="This database contains case law and legal scholarship about CC licenses and the issues that relate to their enforceability and interpretation."/>
+    <meta name="keywords" content="Creative Commons, legal database, case law, CC licenses, legal scholarship, enforceability, legal tools, open access, intellectual property"/>
+    <meta name="author" content="Creative Commons"/>
```

Comment:
Hi, thank you for this. This is a new thing learnt today. Since you mentioned the alphabetical order, i have swapped all three meta tags in the order author, description and keywords. I hope that's okay.